### DISCUSSION TITLE
what's the color theme used in the screenshot from readme?

### DISCUSSION THREAD

**Discussion by haolian9** [magit/transient#218] [Support] [answered]
hi, i'm sorry to be off-topic here, but i am really amazed by the color theme used in this screenshot

![](https://camo.githubusercontent.com/f87497aec74dd0efee4ef78ba2b33b24d5535446b5d5cbef768653f4b945c38c/687474703a2f2f726561646d652e656d6163736169722e6d652f7472616e7369656e742e706e67)

i guess it's "solarized light" alike,  do you mind to share a repo link to it?

**Comment by haolian9** [comment, marked as answer]
never mind, it must be https://github.com/bbatsov/solarized-emacs, i saw it has specific settings for magit.

**Reply by tarsius** [reply]
I forked of 1.3.1 because of some major refactoring I did not like. My version is at https://github.com/tarsiiformes/solarized-theme.

**Reply by haolian9** [reply]
thanks you!

**Reply by tarsius** [reply]
(My version isn't necessarily better. It's just what I am using.)
